src/styles/components/_buttons.scss

```
@@ -0,0 +1,15 @@
+@use '../abstracts' as *;
+
+.btn {
+    padding: pxtoRem(16.2) pxtoRem(31);
+    border-radius: pxtoRem(100);
+    color: $white;
+
+    &--primary-button {
```

```suggestion
    &--primary{
```
You can fix this one in next PR